my-app/src/Pages/Home.tsx

```diff
@@ -0,0 +1,40 @@
+import { useQuery, useQueryClient } from "@tanstack/react-query";
+import { movieService } from "../Services/MovieService";
+import { Link } from "react-router-dom";
+import TruncateMarkup from "react-truncate-markup";
+
+const Home: React.FC<{}> = (props: any) => {
+  const queryClient = useQueryClient();
+  queryClient.invalidateQueries(["movies"]);
+
```

Comment:
If we do this, we are having infinite loop. This queryClient.IncalidateQueries should happen only if some condition is fullfiled. On a press of a button, or something like that